src/main/java/step3/domain/LottoNumber.java

```diff
@@ -0,0 +1,52 @@
+package step3.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+    private static final int LOTTO_START_NUMBER = 1;
+    private static final int LOTTO_END_NUMBER = 45;
+    private static final Map<Integer, LottoNumber> lottoNumberPackage = new HashMap<>();
```

Comment:
상수는 대문자로 표현하는것이 관례입니다
```suggestion
    private static final Map<Integer, LottoNumber> LOTTO_NUMBER_PACKAGE = new HashMap<>();
```

Reply:
캐시를 적용 해주셨네요 💯